feat(modules.clickhouse): Add zookeeper for clickhouse clusterization (#1995)
testcontainers/testcontainers-go: docs/modules/clickhouse.md

[Init script content](../../modules/clickhouse/testdata/init-db.sh)
<!--/codeinclude-->

#### Zookeeper

> If you would like to perform queries against clusterized ClickHouse you need to start zookeeper near your cluster and pass link to it via `config.xml`

Reviewer: ```suggestion
Clusterized ClickHouse requires to start Zookeeper and pass link to it via `config.xml`.
```
